src/cqrs/event/store/AbstractMysqlAggregateEventStore.ts

```diff
@@ -0,0 +1,89 @@
+import { MySQLClient, Autowire, LogManager, DBTransaction, ExtendedError } from 'inceptum';
+import * as moment from 'moment';
+import { LockViolationError } from '../../error/LockViolationError';
+import { AggregateEventStore } from './AggregateEventStore';
+
+export const mySQLDateTimeFormat = 'YYYY-MM-DD HH:mm:ss';
+export const mySQLDateFormat = 'YYYY-MM-DD';
+const logger = LogManager.getLogger(__filename);
+
+interface MysqlAggregateEventData {
+  event_id: string,
+  aggregate_id: string,
+  eventCreatedTime: string,
+  eventCommittedTime: string,
+  eventType: string,
+  eventContent: string,
+}
+
+export abstract class AbstractMysqlAggregateEventStore extends AggregateEventStore {
+  @Autowire('AggregateMySQLClient')
+  private mysqlClient: MySQLClient;
+
+  async getEventsOf(aggregateId: string): Promise<any[]> {
+    const sql = `SELECT *
+                    FROM aggregate_events
+                    WHERE aggregate_id = ?
+                    ORDER BY event_committed_time ASC`;
```

Comment:
Should the order be by ordinal? And should we make sure there's an index on (aggregate_id,ordinal)?